h/static/styles/components/_form.scss

```diff
@@ -82,6 +82,8 @@
 }
 
 .form-description {
+  @include typography.font-big;
```

Comment:
Makes the "form message" text (notification and/or instructions for the user) that's displayed on the account-registration-successful and new-password pages bigger.